site.yml

```diff
@@ -17,6 +17,12 @@ site:
         plugins:
             highlight:
                 $class: Statocles::Plugin::Highlight::RexOps
+            lint:
```

Comment:
I know the synopsis and this section are semantically equivalent, but how about using *exactly* the same content instead?

Reply:
You mean updating the synopsis in the RexOps linter module?

Reply:
yes, I mean updating the synopsis.

Reply:
Fixed in followup commit.